l10n_it_partially_deductible_vat/__openerp__.py

```diff
@@ -24,27 +24,16 @@
 ##############################################################################
 
 {
-    "name" : "Italy - Partially Deductible VAT",
-    "version" : "0.1",
-    "depends" : ['account'],
-    "author" : "OpenERP Italian Community,Odoo Community Association (OCA)",
-    "description": """
-    Calcoli aggiuntivi per la gestione dell'IVA parzialmente detraibile. Senza questo modulo non è possibile calcolare correttamente imponibili, imposte e conti imposte relativi all'IVA parzialmente detraibile.
-    La configurazione dell'IVA parzialmente detraibile deve essere quella utilizzata dal modulo 'l10n_it', anche se non viene usato tale modulo.
-    """,
+    "name": "Italy - Partially Deductible VAT",
+    "version": "1.0.0",
+    "depends": ['account'],
```

Comment:
Ritengo corretto che ```l10n_it_partially_deductible_vat``` non dipenda da ```l10n_it_split_payment``` in quanto le funzionalità sono scorrelate.

Travis sta testando tutti i moduli di l10n-italy insieme:
```
unbuffer coverage run /home/travis/odoo-8.0/openerp-server -d openerp_test --stop-after-init --log-level info --log-handler openerp.tools.yaml_import:DEBUG --test-enable --init account_invoice_entry_date,account_vat_period_end_statement,l10n_it_abicab,l10n_it_account,l10n_it_ateco,l10n_it_base,l10n_it_base_location_geonames_import,l10n_it_fatturapa,l10n_it_fatturapa_out,l10n_it_fiscalcode,l10n_it_ipa,l10n_it_partially_deductible_vat,l10n_it_pec,l10n_it_rea,l10n_it_ricevute_bancarie,l10n_it_split_payment,l10n_it_vat_registries,l10n_it_ddt
```
quindi sembra che le modifiche di questa PR vadano a compromettere il corretto funzionamento di ```l10n_it_split_payment```. Bisognerebbe verificare come mai